### ISSUE TITLE
Integration test assumes ACT support

### ISSUE THREAD

**Issue by joholl** [tpm2-software/tpm2-tss#2186]
The `esys-act-set-timeout.int` test assumes ACT support and fails if the optional command is not implemented on a TPM. It should skip if this command is not supported.

https://github.com/tpm2-software/tpm2-tss/blob/9288970a3e657cdee85d08d3813199ec864de3ad/test/integration/esys-act-set-timeout.int.c#L63-L64

**Comment by JuergenReppSIT** [comment]
@jholl, @AndreasFuchsSIT  The test should already be skipped if `Esys_ACT_SetTimeout` returns `TPM2_RC_COMMAND_CODE`:
```
   /* If the TPM doesn't support it return skip */
    if (r == TPM2_RC_COMMAND_CODE)
        return EXIT_SKIP;
    else
        return EXIT_FAILURE;
}
```

**Comment by AndreasFuchsSIT** [comment]
Very true, I'd say.
@joholl agreed ?

**Comment by joholl** [comment]
Oh sorry guys, I actually wrote an issue about the wrong test o.O

This test skips just fine, as @JuergenReppSIT explained. The respective `Esys_GetCapability` test does not, though:

https://github.com/tpm2-software/tpm2-tss/blob/67a6120a9153a4716987e70080f15eb2f32a976e/test/integration/esys-get-capability-act.int.c#L28-L39

We might want to try calling `Esys_ACT_SetTimeout` before and skip if the command/feature is not supported.

**Comment by AndreasFuchsSIT** [comment]
FIxed in master, will be fixed in 3.2
